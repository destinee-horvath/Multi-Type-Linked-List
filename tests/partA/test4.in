NEW 6
tab

space
 
nothing

VIEW 0
VIEW 100
VIEW-NESTED 100 
REMOVE 0
VIEW 0
NEW 3
a
new
list
VIEW 0
VIEW 1
