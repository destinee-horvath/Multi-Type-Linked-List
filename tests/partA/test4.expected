List 0: tab ->   -> space ->   -> nothing -> 
tab ->   -> space ->   -> nothing -> 
INVALID COMMAND: VIEW
INVALID COMMAND: VIEW-NESTED
List 0 has been removed.

Number of lists: 0
INVALID COMMAND: VIEW
List 1: a -> new -> list
INVALID COMMAND: VIEW
a -> new -> list
